pkg/blob/controllerserver.go

```diff
@@ -679,6 +700,69 @@ func (d *Driver) DeleteBlobContainer(ctx context.Context, subsID, resourceGroupN
 	})
 }
 
+// CopyBlobContainer copies a blob container in the same storage account
+func (d *Driver) CopyBlobContainer(ctx context.Context, req *csi.CreateVolumeRequest, accountKey, dstContainerName, storageEndpointSuffix string, secrets map[string]string) error {
+	var volumeID string
+	if req.GetVolumeContentSource() != nil {
+		if req.GetVolumeContentSource().GetVolume() != nil {
+			volumeID = req.GetVolumeContentSource().GetVolume().GetVolumeId()
+		}
+	}
+	resourceGroupName, accountName, srcContainerName, _, subsID, err := GetContainerInfo(volumeID)
+	if err != nil {
+		return status.Error(codes.NotFound, err.Error())
```

Comment:
just `return err`, don't need to wrap error in sub function since it's already wrapper in outside function

Reply:
done

Reply:
[91m[1m• Failure [0.178 seconds][0m
Controller Service [Controller Server]
[90m/home/prow/go/src/github.com/kubernetes-csi/csi-test/pkg/sanity/tests.go:45[0m
  CreateVolume
  [90m/home/prow/go/src/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:369[0m
    [91m[1mshould fail when the volume source volume is not found [It][0m
    [90m/home/prow/go/src/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:641[0m

    [91munexpected error: error parsing volume id: "fake-vol-id-4a52f688-e", should at least contain two #
    Expected
        <codes.Code>: 2
    to equal
        <codes.Code>: 5[0m

Reply:
It seems that in sanity test it should be NotFound error.